Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.12 as build
+FROM python:3.12-alpine as build
```

Comment:
Just keep it as debian based. Alpine is way too hard to work with, given the size tradeoff. Chances are we will need to install build tools to build the project anyway.

Reply:
Switched back to Debian.